tensorflow_addons/activations/__init__.py

```diff
@@ -22,3 +22,4 @@
 from tensorflow_addons.activations.rrelu import rrelu
 from tensorflow_addons.activations.sparsemax import sparsemax
 from tensorflow_addons.activations.tanhshrink import tanhshrink
+from tensorflow_addons.activations.snake import snake
```

Comment:
place in alphabetical order

Reply:
@WindQAQ @seanpmorgan Just as a reminder can we automate with https://pypi.org/project/flake8-isort/?

Reply:
Looks good to me. Could you open and issue and PR for it? I'm fine to automate it.